phantazm-api/src/main/java/com/github/phantazmnetwork/api/config/ItemStackConfigProcessor.java

```diff
@@ -0,0 +1,39 @@
+package com.github.phantazmnetwork.api.config;
+
+import com.github.steanky.ethylene.core.ConfigElement;
+import com.github.steanky.ethylene.core.ConfigPrimitive;
+import com.github.steanky.ethylene.core.processor.ConfigProcessException;
+import com.github.steanky.ethylene.core.processor.ConfigProcessor;
+import net.minestom.server.item.ItemStack;
+import org.jetbrains.annotations.NotNull;
+import org.jglrxavpok.hephaistos.nbt.NBTCompound;
+import org.jglrxavpok.hephaistos.nbt.NBTException;
+import org.jglrxavpok.hephaistos.parser.SNBTParser;
+
+import java.io.StringReader;
+
+public class ItemStackConfigProcessor implements ConfigProcessor<ItemStack> {
+    @SuppressWarnings("UnstableApiUsage")
+    @Override
+    public @NotNull ItemStack dataFromElement(@NotNull ConfigElement element) throws ConfigProcessException {
+        if (!element.isString()) {
+            throw new ConfigProcessException("itemstack is not string");
```

Comment:
nitpick but might make more sense to say "Element is not a string", since in this case it's not actually an itemstack